docs/repo-docs/crafting-your-repository/configuring-tasks.mdx

```diff
@@ -215,6 +215,8 @@ Below are a few examples of outputs for common tools:
 
 For more on building globbing patterns for the `outputs` key, see [the globbing specification](/repo/docs/reference/globs).
 
+The `outputs` key is optional, see [the API reference](/repo/docs/reference/configuration#outputs) for how Turborepo behaves in this case.
```

Comment:
I prefer redirecting the reader to the accurate reference, instead of explaining here. This avoid duplication, and confusion for when docs are not aligned, or doesn't tell the whole story.

Reply:
The reference and the how-to (Crafting your Repository) serve different purposes. The reference is terse, rote, and dry, while the how-to is more introductory, and caters to a different type of a reader.

This is good information to add, though it might be better in a different place. See https://github.com/vercel/turborepo/pull/9185/files#r1776190519